homeassistant/external/vera

```diff
@@ -1 +1 @@
-Subproject commit 30c59781d63322db2160ff00a4b99f16ead40b85
+Subproject commit 68c025f893e50d19508ad6b7d3b12d4753482310
```

Comment:
Are we still using sub projects for this and not a pip install from the github repo?